plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py

```diff
@@ -43,10 +45,23 @@
 cur_py_version = (
     f"{sys.version_info.major}.{sys.version_info.minor}.{sys.version_info.micro}"
 )
-ami = os.environ.get("AWS_RAY_AMI", "")
-security_group_id = os.environ.get("AWS_RAY_SECURITY_GROUP", "")
-subnet_id = os.environ.get("AWS_RAY_SUBNET", "")
-instance_role = os.environ.get("INSTANCE_ROLE_ARN", "")
+
+aws_not_configured = True
+aws_not_configured_msg = "AWS credentials not configured correctly. Skipping AWS tests."
+try:
+    ec2 = boto3.client("ec2")
+    ec2.describe_regions()
+    aws_not_configured = False
+except (NoCredentialsError, NoRegionError):
+    pass
```

Comment:
Style wise, I would go:
```suggestion
try:
    ec2 = boto3.client("ec2")
    ec2.describe_regions()
    aws_not_configured = False
except (NoCredentialsError, NoRegionError):
    aws_not_configured = True
    aws_not_configured_msg = "AWS credentials not configured correctly. Skipping AWS tests."
```